### PR TITLE
feat: playground support promql

### PR DESCRIPTION
**Pull request by alili**

### CODE REVIEW

**Review by ZonaHex — COMMENTED** [review]
image
Use PromQL logs

**Comment by ZonaHex** [comment]
image
Start and end time is not showing properly.

**Comment by ZonaHex** [comment]
Add default absolute time range for playground

File: src/components/markdown-render/components/code-editor.vue [hunk 1]
```diff
@@ -156,6 +174,13 @@
     // todo: refresh tables data and when
   }
 
+  const copy = () => {
+    navigator.clipboard.writeText(code.value)
+    Message.success({
+      content: i18n.global.t('copied'),
+      duration: 2 * 1000,
+    })
+  }
```
Comment (ZonaHex) [inline src/components/markdown-render/components/code-editor.vue:183]: Can this be like the copy icon UI (change icon after copied)? The global message is difficult for users to notice.

File: src/components/markdown-render/components/importPresets.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+      Message.success({
+        content: i18n.global.t('playground.import', {
+          lines: res.output[0].affectedrows,
+          time: res.networkTime,
+          duration: 5 * 1000,
+        }),
+      })
+    } catch (error) {
+      console.log(`error:`, error)
```
Comment (ZonaHex) [inline src/components/markdown-render/components/importPresets.vue:38]: Is this necessary?

File: src/components/markdown-render/components/code-editor.vue (hunk 1 above)
Comment (alili) [reply src/components/markdown-render/components/code-editor.vue:183]: Not very easy to do, arco does not provide a simple interface to do so

File: src/store/modules/code-run/index.ts [hunk 3]
```diff
@@ -49,12 +47,15 @@ const useCodeRunStore = defineStore('codeRun', () => {
...
     try {
       // TODO: try something better
       let oneResult = {} as ResultType
-      const res: HttpResponse = await API_MAP[type](codeInfo)
-
+      const res: HttpResponse = await API_MAP[type](codeInfo, params)
+      Message.success({
+        content: i18n.global.t('dashboard.runSuccess'),
+        duration: 2 * 1000,
+      })
```
Comment (ZonaHex) [inline src/store/modules/code-run/index.ts:58]: We already have success messages.

**Review by ZonaHex — APPROVED** [review]

**Pull request merged**